src/app/reports/menu/menu.html

```diff
@@ -11,12 +11,7 @@
       <span>Utility</span>
     </h4>
   </a>
-  <a ui-sref="reports.layout.facility" ui-sref-active="active" class="list-group-item">
```

Comment:
So https://dev-dd.ehealth.org.ng/#/reports/facility is duplicated with https://dev-dd.ehealth.org.ng/#/facility/faulty? Those views are showing different results. Maybe removing it should be left for a separate PR?